<?xml version="1.0"?>
<launch>
<rosparam command="load" file="$(find beginner_tutorials)/params/params.yaml"/>
<node name="solar_system" pkg="beginner_tutorials" type="solarsystem.py" output = "screen"/>
</launch>